packages/remediations/src/RemediationWizard/RemediationWizard.js

```diff
@@ -159,7 +180,20 @@ RemediationWizard.propTypes = {
         systems: propTypes.arrayOf(propTypes.string),
         onRemediationCreated: propTypes.func
     }).isRequired,
-    basePath: propTypes.string
+    basePath: propTypes.string,
+    registry: propTypes.object.isRequired
```

Comment:
ReducerRegistry is not `object` but class. Use `PropTypes.instanceOf(ReducerRegistry)`